examples/01_file_generator/runner.py

```diff
@@ -55,5 +55,5 @@
 fuzzer.set_target(target)
 fuzzer.start()
 print('-------------- done with fuzzing -----------------')
-raw_input('press enter to exit')
+input('press enter to exit')
```

Comment:
replace with six.input(), input() raises an IOError on py2 if an empty line is provided.